src/client/src/utils/constants.ts

```diff
@@ -30,6 +30,9 @@ const INTL_MESSAGES = defineMessages({
   }
 });
 
+const BOOTSTRAP_LICENSE =
+  "https://github.com/twbs/bootstrap/blob/master/LICENSE";
```

Comment:
is there a reason this is not with the `const WEB_TEMPLATE_STUDIO_LINKS`?

Reply:
`WEB_TEMPLATE_STUDIO_LINKS` include the links that are related to the  WebTS repository. However, this is a license link for bootstrap. If more license links like these need to be included in future, we could group them under a licenses object.